File: Swift/assign-cookies.swift
class Solution {
    func findContentChildren(_ g: [Int], _ s: [Int]) -> Int {
        var g = g.sorted()
        var s = s.sorted()

        var res = 0
        var i = 0

        for j in 0..<s.count {
            if i == g.count {
                break
            }
            if s[j] >= g[i] {
                res += 1
                i += 1
            }
        }

        return res
    }
}

File: Python/assign-cookies.py
class Solution:
    def findContentChildren(self, g, s):
        """
        :type g: List[int]
        :type s: List[int]
        :rtype: int
        """
        s.sort()
        g.sort()
        res = 0
        i = 0

        for j in range(len(s)):
            if i == len(g):
                break
            if s[j] >= g[i]:
                res += 1
                i += 1

        return res
